src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -53,12 +54,13 @@ class BankAccountStep extends React.Component {
         this.toggleTerms = this.toggleTerms.bind(this);
         this.addManualAccount = this.addManualAccount.bind(this);
         this.addPlaidAccount = this.addPlaidAccount.bind(this);
+        this.debouncedUpdateReimbursementAccountDraft = _.debounce(this.debouncedUpdateReimbursementAccountDraft.bind(this), 100, false);
         this.state = {
             // One of CONST.BANK_ACCOUNT.SETUP_TYPE
             bankAccountAddMethod: props.achData.subStep || undefined,
-            hasAcceptedTerms: props.achData.acceptTerms || true,
-            routingNumber: props.achData.routingNumber || '',
-            accountNumber: props.achData.accountNumber || '',
+            hasAcceptedTerms: lodashGet(props, ['reimbursementAccountDraft', 'hasAcceptedTerms']) || lodashGet(props, ['achData', 'acceptTerms'], true),
+            routingNumber: lodashGet(props, ['reimbursementAccountDraft', 'routingNumber']) || lodashGet(props, ['achData', 'routingNumber'], ''),
+            accountNumber: lodashGet(props, ['reimbursementAccountDraft', 'accountNumber']) || lodashGet(props, ['achData', 'accountNumber'], ''),
```

Comment:
Thoughts on DRY-ing this up by enforcing use of the same field names and something like ?

```js
getDefaultStateForField(fieldName, default) {
    return lodashGet(props, ['reimbursementAccountDraft', fieldName]) 
        || lodashGet(props, ['achData', fieldName], default)
}
```

Unsure if this is worth it as we have `hasAcceptedTerms` in one spot and `acceptTerms` in another spot, but _could_ also clean that up.

We could pass `props` as well and use it in all of these forms.

Reply:
Thanks for this suggestion!